File: web/js/shinow/app/model/Provider.js
/**
 * <p>
 * 供应商信息表
 * </p>
 * 
 * @author 代码生成器
 * @class app.model.Provider
 * @extends Ext.data.Model
 */
Ext.define('app.model.Provider', {
		extend : 'Ext.data.Model',
		alias : 'model.Provider',
		fields : [
				{
					name : 'providerCode',
					type: 'string'
				},
				{
					name : 'providerName',
					type: 'string'
				},
				{
					name : 'providerNameAb',
					type: 'string'
				},
				{
					name : 'address',
					type: 'string'
				},
				{
					name : 'linkName',
					type: 'string'
				},
				{
					name : 'linkTel',
					type: 'string'
				},
				{
					name : 'qq',
					type: 'string'
				},
				{
					name : 'email',
					type: 'string'
				},
				{
					name : 'sortId',
					type: 'int'
				},
				{
					name : 'status',
					type: 'string'
				}
		]
});

File: src/main/java/com/shinowit/entity/Provider.java
package com.shinowit.entity;

import java.io.Serializable;

public class Provider implements Serializable {
    private String providerCode;

    private String providerName;

    private String providerNameAb;

    private String address;

    private String linkName;

    private String linkTel;

    private String qq;

    private String email;

    private Integer sortId;

    private Boolean status;

    private static final long serialVersionUID = 1L;

    public String getProviderCode() {
        return providerCode;
    }

    public void setProviderCode(String providerCode) {
        this.providerCode = providerCode;
    }

    public String getProviderName() {
        return providerName;
    }

    public void setProviderName(String providerName) {
        this.providerName = providerName;
    }

    public String getProviderNameAb() {
        return providerNameAb;
    }

    public void setProviderNameAb(String providerNameAb) {
        this.providerNameAb = providerNameAb;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getLinkName() {
        return linkName;
    }

    public void setLinkName(String linkName) {
        this.linkName = linkName;
    }

    public String getLinkTel() {
        return linkTel;
    }

    public void setLinkTel(String linkTel) {
        this.linkTel = linkTel;
    }

    public String getQq() {
        return qq;
    }

    public void setQq(String qq) {
        this.qq = qq;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public Integer getSortId() {
        return sortId;
    }

    public void setSortId(Integer sortId) {
        this.sortId = sortId;
    }

    public Boolean getStatus() {
        return status;
    }

    public void setStatus(Boolean status) {
        this.status = status;
    }
}